fuzzers/baby_fuzzer/src/main.rs

```diff
@@ -36,7 +39,12 @@ pub fn main() {
             if buf.len() > 1 && buf[1] == b'b' {
                 signals_set(2);
                 if buf.len() > 2 && buf[2] == b'c' {
-                    panic!("=)");
+                    unsafe {
+                        #[cfg(unix)]
+                        panic!("=(");
+                        #[cfg(windows)]
+                        write_volatile(0 as *mut u32, 0);
```

Comment:
Can we find a more general fix that can also catch `panic`? Maybe similar to what winafl does?

Reply:
I looked into winafl's code, 
https://github.com/googleprojectzero/winafl/blob/ea5f6b85572980bb2cf636910f622f36906940aa/winafl.c#L728
as far as I know, it seems to use dynamoRIO's feature to wrap windows's internal functions, which are not applicable in this case.

Reply:
Let's add a comment about that in the example baby fuzzer with a link to that source line, so we can fix it later